Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * Commandline Interface: Return exit code ``2`` on uncaught exceptions.
  * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
+ * EVM: Basic support for the EVM version "Paris".
```

Comment:
I see we did not add changelog for introducing some recent versions (London, Berlin, Istanbul), only for making them default. We used to do that with earlier versions though.

Reply:
We can adjust the changelog entry to ``Support for paris`` anyways, once we have the ``prevrandao`` change (which will be before the next release anyways).